staging/src/k8s.io/apiserver/pkg/server/patch_genericapiserver.go

```diff
@@ -143,3 +145,7 @@ func isLocal(req *http.Request) bool {
 
 	return false
 }
+
+func isKubeApiserverLoopBack(req *http.Request) bool {
```

Comment:
based on https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-remote-libvirt-s390x-4.6/1308979535697416192/artifacts/e2e-remote-libvirt/pods/openshift-kube-apiserver_kube-apiserver-ci-op-tjzfzjcn-486b5-lzwlc-master-1_kube-apiserver.log

at least the following operators are also using loopback:
- `cluster-kube-controller-manager-operator`
- `cluster-kube-apiserver-operator`


do you know why?
would we like to put them on a white list?

Reply:
that's a follow-up.

It is the DR sidecar afaik. But before whitelisting that we have to change the user agent there. We do not want to whitelist kcm-o or ka-o as a whole.